internal/servers/worker/status.go

```diff
@@ -16,7 +16,7 @@ import (
 // In the future we could make this configurable
 const (
 	statusInterval           = 2 * time.Second
-	statusTimeout            = 10 * time.Second
+	statusTimeout            = 3 * time.Second
```

Comment:
This makes me a bit nervous. I wonder if a heavily loaded DB might take more than 3 seconds to return a reply, especially if there are large numbers of sessions and only a single worker.

Reply:
This was discussed internally, I did check in, I can forward the discussion if you like. Let me know!

Reply:
Discussed internally to change the timeout to 5 seconds (will be done shortly).